src/overlays/actors/ovl_Bg_Haka_Gate/z_bg_haka_gate.c

```diff
@@ -10,31 +10,53 @@
 
 #define THIS ((BgHakaGate*)thisx)
 
+#define vTimer actionVar1
+#define vTurnDirection actionVar1
+
+#define vTurnRateDeg10 actionVar2
+#define vOpenAngle actionVar2
+
+#define vTurnAngleDeg10 actionVar3
+#define vFlameScale actionVar3
+
+#define vIsSkullOfTruth actionVar4
+#define vRotYdeg10 actionVar4
+
+#define vInitTurnAngle actionVar5
+#define vScrollTimer actionVar5
```

Comment:
I think it would be better to group these by the context they should be used in, instead of grouping them by 1/2/3/4/5